### PR TITLE
Recipe for Elasticsearch

### PR DESCRIPTION
**Pull request by pmlandwehr**
Breaking for unclear reasons - seems to be skipping all build steps when run locally.

### CODE REVIEW

File: recipes/elasticsearch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - coverage
+    - mock
+    - nose
+    - nosexcover
+    - pyaml
+    - requests >=1.0.0,<3.0.0
+
+about:
+  home: https://github.com/elastic/elasticsearch-py
+  license: Apache Software License
```
Comment (pmlandwehr) [inline recipes/elasticsearch/meta.yaml]: Need to update this to Apache v. 2, can do this after the build

**Comment by pmlandwehr** [comment]
See #813 - `elasticsearch` wants urllib3 before it builds correctly.

**Comment by pmlandwehr** [comment]
Forcing a rebuild, since the urllib3 distribution on conda-forge is now building for windows.

**Comment by pmlandwehr** [comment]
(ping @ocefpaf)

File: recipes/elasticsearch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  #   - coverage
+  #   - mock
+  #   - nose
+  #   - nosexcover
+  #   - pyaml
+  #   - requests >=1.0.0,<3.0.0
+
+about:
+  home: https://github.com/elastic/elasticsearch-py
+  license: Apache Software License Version 2.0
```
Comment (jakirkham) [inline recipes/elasticsearch/meta.yaml]: "Apache 2.0" should be sufficient.

**Comment by jakirkham** [comment]
One comment. Otherwise LGTM.

File: recipes/elasticsearch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - urllib3 >=1.8,<2.0
+
+test:
+  imports:
+    - elasticsearch
+    - elasticsearch.client
+    - elasticsearch.connection
+    - elasticsearch.helpers
+
+  # requires:
```
Comment (ocefpaf) [inline recipes/elasticsearch/meta.yaml]: Can you remove all these comments?

**Comment by pmlandwehr** [comment]
Should be ready to go after this rebuild.

**Pull request merged**